components/private_channel/browser/private_channel_service.cc

```diff
@@ -0,0 +1,320 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
```

Comment:
wrong year

Reply:
also filename should be `private_channel.cc` to match the class name

Reply:
- [ ] change file name to `private_channel.cc`
- [ ] update year